avalon/tools/models.py

```diff
@@ -6,6 +6,7 @@
 from ..vendor import qtawesome
 from .. import io
 from .. import style
+from .. import lib
 
```

Comment:
Not tested yet, but based on the changes down below, I think this relative import should change to `from . import lib` ☺️

Reply:
That's because I though I put `project_use_silo` into `avalon.lib`, but i guess `avalon.tools.lib` is better position since is used only in tools :)